site/content/docs/user/resources.md

```diff
@@ -71,6 +72,9 @@ https://mauilion.dev/posts/kind-k8s-testing/
 
 https://projectcontour.io/kindly-running-contour/
 
+### Local Ingress Domains for your Kind Cluster
+
+- OSX: https://mjpitz.com/blog/2020/10/21/local-ingress-domains-kind/
```

Comment:
this does not render ok, you can see it here

https://deploy-preview-1908--k8s-kind.netlify.app/docs/user/resources/

maybe just remove the `- OSX:` part and add only the url, so it is consistent with the previous sections ... or add the OSX part to the section title if you want to highlight the OS